src/Raven.Studio/typescript/components/pages/database/tasks/panels/SubscriptionPanel.tsx

```diff
@@ -180,8 +180,8 @@ function ConnectedClients(props: ConnectedClientsProps) {
             )}
             <div className="connected-clients-items">
                 {connections.Results.length === 0 && <div>No clients connected</div>}
-                {connections.Results.map((connection) => (
-                    <div className="text-center py-2 connected-client">
+                {connections.Results.map((connection, index) => (
+                    <div key={"connectedClient" + index} className="text-center py-2 connected-client">
```

Comment:
let's use connection.ClientUri here for key generation. 

See: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md